public/index.html

```diff
@@ -0,0 +1,10 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="utf-8" />
+    <title>React App</title>
```

Comment:
title이라면 이게 자연스러워보여요
```suggestion
    <title>Link Library</title>
```